src/pages/map.jsx

```
@@ -1,50 +1,63 @@
-import { MapContainer, TileLayer, Marker, Popup, ScaleControl, useMap } from 'react-leaflet';
 import L from 'leaflet';
 import { useEffect, useState, useMemo, useCallback } from 'react';
+import { MapContainer, TileLayer, Marker, Popup, ScaleControl, useMap } from 'react-leaflet';
```

ESLint

[error] There should be no empty line within import group

(import/order)